src/routes/AppRouter/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { Suspense } from "react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
```

Comment:
Un warning 'Route' is defined but never used.

Reply:
Solucionadoo